.ci/esy-build-steps.yml

```diff
@@ -16,8 +16,8 @@ steps:
   - script: 'npm install'
     displayName: 'npm install'
   - ${{ if eq(parameters.platform,  'Ubuntu_16_04') }}:
-    - script: npm i -g esy@0.6.4
-      displayName: 'Install esy@0.6.4 for Ubuntu 16.04 (bootstrapping)'
+    - script: npm i -g esy@0.6.7
```

Comment:
I think this will fail. 0.6.4 was the last version of esy that works on 16.04 I'm told

Reply:
yeah reverting it until we release the next esy version using static binaries